website/content/gateway/api_reference/resources/authorizations/authorizations.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "authorizations"
+date: 2022-04-13T12:37:22+02:00
+anchor: "authorizations"
+weight: 105
+---
+#### Authorizations
```

Comment:
When reading this rendered (locally) I had the thought that I would love to change the vertical spacing a bit ...

See the red marks/squares:

![Screenshot from 2022-09-05 11-15-29](https://user-images.githubusercontent.com/14069221/188414846-1d4c3b43-2da3-4e39-bb1c-c04808cad964.png)

Is it possible to decrease the vertical spacing in those places? Perhaps it just me. I don't know. For reference, let me show the old layout:

![Screenshot from 2022-09-05 11-20-35](https://user-images.githubusercontent.com/14069221/188415200-6a66d7bb-71d0-4d1f-89f1-b0088f329657.png)

Reply:
Regarding the extra space `hugo` seems to add an extra `<p></p>` tag. And I don't not know why.
![image](https://user-images.githubusercontent.com/3591989/190494500-372fa3ee-56cc-4665-af66-aedb17210384.png)

Reply:
As mentioned in https://github.com/clearhaus/gateway-api-docs/pull/151#issuecomment-1248629139, the most recent commit doesn't work, so I had to do `apt update && apt install nodejs npm -y && npm i -g yarn` before `./build/run.sh` in the container to test this.

However, I'm not able to reproduce the bug. There's no `<p></p>` between `<h5 id="parameters">Parameters</h5>` and `<dl>...</dl>` below it when testing locally.

Are you sure you have a clean working directory?